client/src/components/application/NameExamination.vue

```diff
@@ -82,6 +82,9 @@
       userId() {
         return this.$store.getters.userId;
       },
+      userCanEdit() {
```

Comment:
THis can also be removed since we won't be using userCanEdit in this NameExamination.vue

Reply:
Got it